tron/mesos.py

```diff
@@ -362,6 +383,23 @@ def submit(self, task):
 
         mesos_task_id = task.get_mesos_id()
         self.tasks[mesos_task_id] = task
+        env = task.get_config()['environment']
+        clusterman_resource_str = env.get('CLUSTERMAN_RESOURCES')
+        clusterman_metrics, _ = get_clusterman_metrics()
+        if clusterman_resource_str and clusterman_metrics:
+            clusterman_resources = json.loads(clusterman_resource_str)
+            aws_region = env.get('EXECUTOR_AWS_REGION')
+            cluster = env.get('EXECUTOR_CLUSTER', env.get('PAASTA_CLUSTER'))
+            pool = env.get('EXECUTOR_POOL', env.get('PAASTA_POOL'))
+            metrics_client = clusterman_metrics.ClustermanMetricsBotoClient(
+                region_name=aws_region,
+                app_identifier=f'tron_{cluster}_{pool}',
```

Comment:
I _think_ you need to set `app_identifier` to just `pool`?  I think otherwise clusterman won't be able to find the metrics.

Reply:
oh you're right.  that feels like an odd design decision... what happens when two clusters have the same pool name?

Reply:
I filed CLUSTERMAN-327 about this long ago, but we never did anything about it.  I guess it hasn't hurt anything yet, but seems like it could cause confusing (and maybe even hard-to-notice) issues.